packages/other/src/PropertyEditor.css

```diff
@@ -82,6 +82,10 @@
 .other_PropertyEditor .property-table tbody > tr.disabled > td {
     color:gray;
 }
+.other_PropertyEditor .property-table tbody > tr.invalid > td {
+    color:red;
+    background-color: rgba(255,0,0,0.2);
```

Comment:
I added this line via the github "Change this file using the online editor" feature.